### PR TITLE
testing prism returns external file

### PR DESCRIPTION
**Pull request by ewan-IW**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@ __pycache__/
 .idea/
 .venv/
 
+jwtRS512.key
```
Comment (artronics) [inline .gitignore:20]: please add `**/*.key`. This will prevent git from tracking any private key accidentally.

File: openapitools.json
Comment (artronics) [inline openapitools.json:1]: This file should be `.gitignored` as well. Also please remove it from history: `git rm openapitools.josn`

File: tests/test_endpoints.py [hunk 2]
```diff
@@ -116,3 +117,32 @@ def test_nhs_login_p9(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
...
+        "Interaction-ID": "urn:nhs:names:services:gpconnect:fhir:operation:gpc.getstructuredrecord-1"
+    }
+    headers.update(nhsd_apim_auth_headers)
+
+    resp = requests.get(
+        f"{nhsd_apim_proxy_url}/Appointment/1",
+        headers=headers
+    )
+
+    expected_response = load_example("../specification/examples/single_appointment.yaml")
```
Comment (artronics) [inline tests/test_endpoints.py:147]: the `path` that you're passing is already relative to the `specification` directory. So this should be `examples/single_appointment.yaml`.

Comment (ewan-IW) [reply tests/test_endpoints.py:147]: But its from the tests directory not the specification one ?

**Review by artronics — APPROVED** [review]

**Pull request merged**